I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/storyscript/secchpt-green.rpy", line 526, in script
    show but temp at center with begone
Exception: Parameter 'new_widget' is not known by ATL Transform.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/storyscript/secchpt-green.rpy", line 526, in script
    show but temp at center with begone
  File "C:\Users\J\Documents\renpy-7.1.3-sdk\renpy\ast.py", line 1356, in execute
    renpy.exports.with_statement(trans, paired)
  File "C:\Users\J\Documents\renpy-7.1.3-sdk\renpy\exports.py", line 1481, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "C:\Users\J\Documents\renpy-7.1.3-sdk\renpy\display\core.py", line 2227, in do_with
    clear=clear)
  File "C:\Users\J\Documents\renpy-7.1.3-sdk\renpy\display\core.py", line 2672, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "C:\Users\J\Documents\renpy-7.1.3-sdk\renpy\display\core.py", line 3012, in interact_core
    new_widget=layers_root)
  File "C:\Users\J\Documents\renpy-7.1.3-sdk\renpy\atl.py", line 433, in __call__
    raise Exception('Parameter %r is not known by ATL Transform.' % k)
Exception: Parameter 'new_widget' is not known by ATL Transform.

Windows-8-6.2.9200
Ren'Py 7.1.3.1092
SpIn 1.0
Sat Apr 13 16:35:12 2019
